src/python/bot/minimizer/utils.py

```diff
@@ -130,9 +131,9 @@ def single_test_run(test_path):
   return True
 
 
-def tokenize(data, level=0):
```

Comment:
Is this function actually referenced anywhere? This was intended as a lightweight simple test, and I think the new tokenizer is probably overkill for what I intended here. If we can just delete it that would be ideal, but if it's still referenced in any of the main functions for minimizers it's fine to leave it as-is.

Reply:
https://github.com/google/clusterfuzz/blob/3722e18b8f266a5de9d4baa068fb2aab58f4501b/src/python/bot/minimizer/chunk_minimizer.py#L53

Reply:
Thanks for checking. Seems fine as-is, then.